src/profile/v100/entity/account/appearance_id.rs

```diff
@@ -110,59 +108,68 @@ impl From<AppearanceID> for u8 {
 #[cfg(test)]
 mod tests {
 
-    use crate::prelude::*;
+    use super::*;
+
+    #[allow(clippy::upper_case_acronyms)]
+    type SUT = AppearanceID;
```

Comment:
@jakub-rdx @micbakos-rdx @GhenadieVP When I started in November I did not use the `SUT` pattern, but I've been moving over to it gradually. So if you ever see a `mod tests` which does NOT use `type SUT =`, please help convert to it.  `SUT` is great since it makes writing macros much easier to!

Also I did not know I could use `use super::*;`, so if you see ` use crate::prelude::*;` in mod tests, please change to `use super::*;`